### PR TITLE
update coastsat_package to 0.1.46

### PR DESCRIPTION
**Pull request by 2320sharon**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@2320sharon in conda-forge we only do reviews for packages when the PR submitter requests one. That is b/c, most of the time, it is fine to self-merge b/c the changes are package metadata boilerplate most of the time.

TL;DR feel free to merge this one or, if you want, ping me and I can do that.
